lib/model/app_model/list_invoices.dart

```diff
@@ -13,14 +13,14 @@ class AppListInvoices {
       var appInvoices = invoices.map((invoice) {
         var temp = AppInvoice.fromJSON(invoice,
             snackCase: snackCase, isObject: isObject);
-        if (temp.status == status) {
+        if (temp.status == "unpaid") {
```

Comment:
```suggestion
        if (temp.status == "unpaid") {
```

why? this is wrong `AppListInvoices` need to have a status

Reply:
I accidentally changed that to unpaid. I have fixed it in the later commit.